disperser/metrics.go

```diff
@@ -28,6 +28,13 @@ type Metrics struct {
 	logger   common.Logger
 }
 
+// The error space of dispersal requests.
+const (
+	StoreBlobFailure          string = "store-blob-failed"   // Fail to store the blob to S3
```

Comment:
Think it's possible for the blob to get stored in S3 but the method fails due to not queuing the blob on Dynamo.